temperature/schemas.py

```diff
@@ -0,0 +1,20 @@
+from datetime import datetime
+
+from pydantic import BaseModel
+
+
+class TemperatureBase(BaseModel):
+    date_time: datetime
+    temperature: float
+
+    class Config:
+        orm_mode = True
+
```

Comment:
(https://docs.pydantic.dev/latest/migration/#changes-to-config)

The following config settings have been renamed:

orm_mode → from_attributes